### DISCUSSION TITLE
Trying to get webhook to announce the current join code would like some help.

### DISCUSSION THREAD

**Discussion by SLYROOKO** [lloesche/valheim-server-docker#621] [Q&A]
My current relevant env vars are:

VALHEIM_LOG_FILTER_CONTAINS_JOINCODE=join code
ON_VALHEIM_LOG_FILTER_CONTAINS_JOINCODE={ read l; l=${l:30 }; curl -sfSL -X POST -H "Content-Type: application/json" -d "{\"username\":\"valheim\",\"content\":\"$l\"}" $DISCORD_WEBHOOK; }

I would like the discord output to be something like:

Session "test" with join code 555555
OR
Created new join code 555555 for session "test" 

not skilled in bash scripting and i'm stuck. i think one attempt where i put l=${l:0:} errored because i didn't escape the quotation marks in the session name.

**Comment by SLYROOKO** [comment]
even just extracting the join code would work as a solution. i was thinking that doing ON_VALHEIM_LOG_FILTER_REGEXP=^[0-9]{6,6}$ might be a better solution.

**Reply by ArrowPrint** [reply]
Any luck on this?
